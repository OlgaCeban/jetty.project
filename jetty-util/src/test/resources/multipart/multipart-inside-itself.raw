--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x
Content-Disposition: form-data; name="reporter"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

<user@company.com>
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x
Content-Disposition: form-data; name="timestamp"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

2018-03-21T18:52:18+00:00
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x
Content-Disposition: form-data; name="comments"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

this couldn't be parsed
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x
Content-Disposition: form-data; name="attachment"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGj
Content-Disposition: form-data; name="fruit"

banana
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGj
Content-Disposition: form-data; name="color"

yellow
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGj
Content-Disposition: form-data; name="cost"

$0.12 USG
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGj
Content-Disposition: form-data; name="comments"

--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x
--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGj--

--kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x--
