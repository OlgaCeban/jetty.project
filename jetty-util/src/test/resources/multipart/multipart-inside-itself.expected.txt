Content-Type|multipart/form-data; boundary="kCFSwstIsIZkZtQ5JM7193kJYcK5WkyvoxsGjx5eCuVFbaeUZ9oK57JCp_p5wP86S5nY4re5x"
Parts-Count|4
Part-ContainsContents|reporter|<user@company.com>
Part-ContainsContents|timestamp|2018-03-21T18:52:18+00:00
Part-ContainsContents|comments|this couldn't be parsed
Part-ContainsContents|attachment|banana